src/TwigComponent/doc/index.rst

```diff
@@ -447,15 +447,25 @@ need to populate, you can render it with:
 Mounting Data
 -------------
 
-Most of the time, you will create public properties and then pass values
-to those as "props" when rendering. But there are several hooks in case
-you need to do something more complex.
+Most of the time, you will create public properties and pass values to them
+as "props" when rendering the component. However, there are several hooks
+available when you need to perform more complex logic.
 
 The mount() Method
 ~~~~~~~~~~~~~~~~~~
 
-For more control over how your "props" are handled, you can create a method
-called ``mount()``::
+The ``mount()`` method gives you more control over how your "props" are handled.
+It is called only once: immediately after your component is instantiated.
+The ``mount()`` method **cannot access the values of the public properties**
+passed when rendering the component.
```

Comment:
I see what you mean, but this is _wrong_ if I want to be picky:
1. you can access the public properties values, if they are not typed or have a default values
2. speaking about only "public properties values" can be misleading, I feel like we should speak about the "props" too in priority